mlir/test/fusion/e2e/resnet50/mixr-resnet-fusion-case-1.mlir

```diff
@@ -0,0 +1,15 @@
+// RUN: rocmlir-opt -migraphx-to-tosa %s | rocmlir-driver -host-pipeline partition,highlevel -targets %arch | FileCheck %s
+// CHECK: linalg.conv_2d
+// CHECK: rock.conv2d
```

Comment:
Let's go ahead and add `// CHECK: linalg.generic` after the check for `rock.conv2d` just to be sure a fusion did happen.

Reply:
Right, I've added a bit elaborative CHECKS make sure its fused :)